src/site/content/en/blog/hands-on-portals/index.md

```diff
@@ -0,0 +1,203 @@
+
+**Portals** are a new HTML element that can help you keep your front-end simple while allowing seamless navigations with custom transitions. The best part is that Portals are quite easy to work with. Think of them like an iframe in that they allow for embedding, but unlike an iframe they also come with features to directly navigate to their content. Seeing is believing: please first check out what we showcased at Chrome Dev Summit last year.
+
+<div style="width:100%; padding-top: 56.25%; position: relative;">
+  <iframe style="width:100%; height: 100%;position: absolute; top: 50%; left: 50%; transform: translate(-50%,-50%);" src="https://www.youtube.com/embed/Ai4aZ9Jbsys?start=1081" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+</div>
+
+As you saw in the manga viewer demo, the current experience was based on classic navigations where the reader had to wait with a blank screen until the browser finished to render the next chapter. Whereas the improved experience with Portals were pre-rendering the next chapter, triggered an animation on user interaction, and activated the portal to create a seamless experience. The demo didn’t show it (because we used Chrome DevTools mobile simulator 😛) but when the pre-rendered chapter on the bottom left corner fully animated and covered the screen, the URL also switched to the next chapter.
+
+Before Portals, we could have rendered another page using an iframe. We could also have added animations to move the frame around the page. But iframes won’t let you navigate into their content. Portals close this gap, enabling interesting use cases.
+
+## Try out Portals in Chrome Canary
+
+You can try out Portals in Chrome Canary by flipping an experimental flag: [`chrome://flags/#enable-portals`](chrome://flags/#enable-portals). Once Portals are enabled, confirm in DevTools that you have the new shiny `HTMLPortalElement`.
+
+<img class="w-screenshot" src="HTMLPortalElement.png" alt="A screenshot of the DevTools console showing the HTMLPortalElement">
+
+Let’s walk through a basic example.
+
+```javascript
+// Create a portal with the wikipedia page, and embed it
+// (like an iframe). You can also use <portal> tag instead.
+portal = document.createElement('portal');
+portal.src = 'https://en.wikipedia.org/wiki/World_Wide_Web';
+portal.style = '...';
+document.body.appendChild(portal);
+
+// When the user touches the preview (embedded portal):
+// do fancy animation, e.g. expand …
+// and finish by doing the actual transition
+portal.activate();
+```
+
+It’s that simple. Try this code in the DevTools console, the wikipedia page should open up. If you wanted to build something like we showed at Chrome Dev Summit, the following snippet will be of interest.
```

Comment:
Suggestion to bring the sentence about 'If you wanted to build' to come after the image showing the wikipedia page. So that the sentence is coming just before the code.. Also, as previously mentioned, not all users will watch the video from Chrome Dev Summit. If possible, can you introduce the snippet, maybe saying something like:

"The following snippet shows you how to use portals in a way that's similar to the portals demo at the start of this article:"

Reply:
Also, is there a way to break up this code a little bit? You could introduce the parts of it in stages, and then show it all together at the end? A glitch tutorial would really work well here. But I know time is precious!

Reply:
First bit, will work on it! 

Second bit, this, yes, I understand that it's good to have that structure you suggested but given the interest of time, can I please keep it as is? If needed, I could work on it after I/O (need to fly in a bit). Also the interesting thing is that, you can copy paste this whole snippet and paste in the console and it works. I kind of want to keep the unity there.

Reply:
Regarding your first comment, can I change it as

"If you wanted to build something like we showed at Chrome Dev Summit which works just like the demo above, 
the following snippet will be of interest."

And having it after the GIF? 1) this snippet doesn't represent the very first GIF in the article 2) I agree with making it understandable for the people that didn't play the YT video so I added "which works just like the demo above" to also reference the GIF above.